leapp-data.spec

```diff
@@ -27,28 +27,37 @@ Conflicts: %{conflict_dists}
 %install
 mkdir -p %{buildroot}%{_sysconfdir}/leapp/files/vendors.d
 %if 0%{?rhel} < 8
-cp -f vendors.d/* %{buildroot}%{_sysconfdir}/leapp/files/vendors.d/
+	cp -f vendors.d/* %{buildroot}%{_sysconfdir}/leapp/files/vendors.d/
```

Comment:
maybe it's better to use tabs here?

Reply:
I believe that I'm already using tabs for conditionals' indentation, including this one. Am I mistaken?

Reply:
sorry, I meant maybe its better to use spaces instead of tabs

Reply:
Hm, the specfile formatting guidelines I remember reading recommended using hard tabs for indentation.
rpmlint seems fine with it too.
Any specific reason you'd prefer spaces?

Reply:
Just got used to spaces in python. Don't see anything special is mentioned here https://rpm-packaging-guide.github.io/

Of course, it's a well-known holly war and it's up to you what to use: spaces ot tabs. 

P.S.: 😃 https://stackoverflow.blog/2017/06/15/developers-use-spaces-make-money-use-tabs/

Reply:
The only thing that is important is not to mix spaces and tabs indentation in one file. If everything is with tabs it's okay. You could ignore the comment